packages/block-library/src/post-content/block.json

```diff
@@ -6,6 +6,7 @@
 		"postType"
 	],
 	"supports": {
+		"align": true,
```

Comment:
Should this also be added for template part block?

Reply:
1ee4458